### PR TITLE
refactor: no need for runtime.Scheme in ApplyClient

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: deploy/resources.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package resources
```
Comment (alexeykazakov) [inline deploy/resources.go:1]: This is not related to this PR, is it?

**Review by alexeykazakov — APPROVED** [review]

File: deploy/resources.go (hunk 1 above)
Comment (xcoulon) [reply deploy/resources.go:1]: right, my bad, I created this branch on top of an existing one, for something not completed yet 🤦‍♂️

Comment (xcoulon) [reply deploy/resources.go]: removed in 28ec1eb
